### DISCUSSION TITLE
Zooming experience in embedded panels

### DISCUSSION THREAD

**Discussion by acontreras89** [grafana/grafana#54962] [Dashboards, Panels & Visualizations]
When a panel is embedded through an iframe in an external (web) application, users can zoom in by selecting (click and drag) the area they want to inspect. However, there is no way to restore the initial timespan (zoom out) other than refreshing the iframe, leading to a quite poor user experience.

It would be nice to have a zoom out functionality for embedded panels ([some libraries like Dygraphs handle this with a double click](https://dygraphs.com/tests/zoom.html)), or at least restrict zoom operations to prevent the user from unwillingly perform an action which cannot be reverted.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
